packages/inventory-compliance/src/SystemRulesTable.test.js

```diff
@@ -0,0 +1,157 @@
+import React from 'react';
+import { shallow, mount } from 'enzyme';
+import toJson from 'enzyme-to-json';
+import SystemRulesTable from './SystemRulesTable';
+import { IntlProvider } from 'react-intl';
+jest.mock('@redhat-cloud-services/frontend-components-remediations/remediationsApi');
+import * as remediationsApi from '@redhat-cloud-services/frontend-components-remediations/remediationsApi';
+import { SortByDirection } from '@patternfly/react-table';
+import { TITLE_COLUMN } from './Constants';
+
+const windowInsightsSpy = {
+    chrome: {
+        auth: {
+            getUser: () => {
+                return { then: (callback) => callback() };
+            }
+        }
+    }
+};
+
+const remediationsResponse = {
+    'ssg:rhel7|pci-dss|xccdf_org.ssgproject.content_rule_docker_storage_configured': false,
+    'ssg:rhel7|pci-dss|xccdf_org.ssgproject.content_rule_service_docker_enabled': false
+};
+
+const system = {
+    id: 'aa9c4497-5707-4233-9e9b-1fded5423ef3',
+    name: '3.example.com',
+    profiles: {},
+    __typename: 'System'
+};
+
+const profileRules = [
+    {
+        system: 'aa9c4497-5707-4233-9e9b-1fded5423ef3',
+        profile: {
+            refId: 'xccdf_org.ssgproject.content_profile_pci-dss',
+            name: 'PCI-DSS v3 Control Baseline for Red Hat Enterprise Linux 7'
+        },
+        rules: [
+            {
+                title: 'Use direct-lvm with the Device Mapper Storage Driver',
+                severity: 'low',
+                rationale: 'foorationale',
+                refId: 'xccdf_org.ssgproject.content_rule_docker_storage_configured',
+                description: 'foodescription',
+                compliant: true,
+                identifier: {
+                    label: 'CCE-80441-9',
+                    system: 'https://nvd.nist.gov/cce/index.cfm',
+                    __typename: 'RuleIdentifier'
+                },
+                references: [],
+                __typename: 'Rule'
+            },
+            {
+                title: 'Enable the Docker service',
+                severity: 'medium',
+                rationale: 'foorationale',
+                refId: 'xccdf_org.ssgproject.content_rule_service_docker_enabled',
+                description: 'foodescription',
+                compliant: true,
+                identifier: {
+                    label: 'CCE-80440-1',
+                    system: 'https://nvd.nist.gov/cce/index.cfm',
+                    __typename: 'RuleIdentifier'
+                },
+                references: [],
+                __typename: 'Rule'
+            }
+        ]
+    }
+];
+
+describe('SystemRulesTable component', () => {
+    it('should render', () => {
+        global.insights = windowInsightsSpy;
+        remediationsApi.getResolutionsBatch.mockImplementation(
```

Comment:
This yields a HTTP get right? If so, use [jest-fetch-mock](https://www.npmjs.com/package/jest-fetch-mock) instead of mocking promises and mocking implementation.